main.tf

```diff
@@ -8,7 +8,7 @@
 // }
 
 # tflint検知用
-// resource "aws_instance" "foo" {
-//   ami           = "ami-0ff8a91507f77f867" #invalid AMI ID. (aws_instance_invalid_ami)
-//   instance_type = "t3.small"
-// }
\ No newline at end of file
+resource "aws_instance" "foo" {
+  ami           = "ami-0ff8a91507f77f867" #invalid AMI ID. (aws_instance_invalid_ami)
```

Comment:
🚫 **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>"ami-0ff8a91507f77f867" is invalid AMI ID.